File: cli/internal/protocol/kinds.go
// Package protocol contém as constantes normativas do protocolo de sync
// (shared/PROTOCOL.md) — espelha app/lib/sync/nostr/kinds.dart. Mudar
// qualquer valor aqui é uma mudança de protocolo coordenada nos dois lados.
package protocol

const (
	TemplateSnapshotKind  = 30078 // NIP-78, addressable
	TemplateChangesetKind = 9411  // TEMPLATE_CHANGESET_KIND
	SnapshotDTag          = "tpl-new-app-snapshot"
	CurrentSchemaVersion  = 1
)

File: app/lib/sync/nostr/kinds.dart
/// Constantes de protocolo compartilhadas com a CLI Go
/// (`cli/internal/protocol/kinds.go`) — fonte normativa em
/// `shared/PROTOCOL.md`. Mudar qualquer valor aqui é uma mudança de
/// protocolo: precisa ser coordenada nos dois lados e registrada lá.
const int templateSnapshotKind = 30078; // NIP-78, addressable
const int templateChangesetKind = 9411; // TEMPLATE_CHANGESET_KIND
const String snapshotDTag = 'tpl-new-app-snapshot';
const int currentSchemaVersion = 1;
